app/src/main/java/org/oppia/app/home/HomeActivity.kt

```diff
@@ -7,7 +7,7 @@ import org.oppia.app.topic.TopicActivity
 import javax.inject.Inject
 
 /** The central activity for all users entering the app. */
-class HomeActivity : InjectableAppCompatActivity(), RouteToExplorationListener, RouteToTopicListener {
+class HomeActivity : InjectableAppCompatActivity(), RouteToExplorationListener, RouteToTopicListener{
```

Comment:
Please fix spacing. Need `` `` before the ``{``.

Reply:
Done.